src/pretix/control/templates/pretixcontrol/order/index.html

```diff
@@ -388,7 +388,7 @@ <h3 class="panel-title">
             </div>
             {% eventsignal event "pretix.control.signals.order_info" order=order request=request %}
             <div class="row payments">
```

Comment:
Is this `row` still needed now? What does the `payments` class do?

Reply:
The .payments class themes the ``dt`` element, see pretix/static/pretixbase/scss/_theme.scss

Reply:
Since the subelements still use ``col``, even though it's ``col-md-12``, we should keep the ``row`` as well, although  we could drop both, probably.